### ISSUE TITLE
Lengths of beams and hole spacing from ends missing

### ISSUE THREAD

**Issue by jonathandann** [villagekit/gridbeam.xyz#16]
Hello, the documentation includes the hole spacing, but not the distance from the first/last hole (I assume center of the hole) to the end of the beam.

can you include this please? I assume it’s half hole-spacing, but unclear.

can you clarify if hole spacing is measured center to center or between the edge of hole circumference(s).

thanks!

**Comment by ahdinosaur** [comment]
> the distance from the first/last hole (I assume center of the hole) to the end of the beam. ... I assume it’s half hole-spacing, but unclear.

yeah, you're right, it's half the hole spacing. so for the original imperial beams it's 1½ inch between holes, and ¾ inch from the first/last hole to the end of the beam. for the metric beams we're doing at [gridkit.nz](https://gridkit.nz) it's 40mm between holes and 20mm from the first/last hole to the end of the beam.

> can you clarify if hole spacing is measured center to center or between the edge of hole circumference(s).

with regards to hole spacing, holes are always measured from their center.

hope that helps, am keen to re-do the gridbeam.xyz website altogether, but for now happy to answer any questions. also have a forum at [discuss.villagekit.com](https://discuss.villagekit.com/) in case you wanna discuss anything grid beam. cheers :purple_heart:
